android/gradle.properties

```diff
@@ -26,6 +26,7 @@ android.enableJetifier=true
 
 # Increase storage capacity (the default is 6 MB)
 AsyncStorage_db_size_in_MB=10
+AsyncStorage_useNextStorage=true
```

Comment:
The changes to this file are not require, they enable the `next` AsyncStorage version 
https://react-native-async-storage.github.io/async-storage/docs/advanced/next